templates/bin/fastcheck

```diff
@@ -19,8 +19,6 @@ bundle exec scss-lint `find app/assets/stylesheets/ -name "*.scss"`
 # new SASS linter
 yarn sass-lint -v --no-exit app/javascripts/**/*.scss
 
-bundle exec slim-lint app/views/ -c .slim-lint.yml
```

Comment:
can we have the erb alternative? or will you tackle that in a separate PR?

Reply:
aha the erb linter. Yes will add it here, thanks for the reminder

Reply:
Yes, please have a look at https://github.com/Shopify/erb-lint